### PR TITLE
Add needsUpgrade 

### PR DESCRIPTION
**Pull request by DG12**
If URL includes A# tell them there is an upgrade available.
Perhaps the suffix was missing a minute ago.
svg/fallback/now-available.png now included.

### CODE REVIEW

**Comment by DG12** [comment]
I also added the svg/fallback/available.png file which was missing but I don't see it included in this Pull Request!

File: css/ruu-vi-styles.css [hunk 1]
```diff
@@ -212,7 +218,7 @@ p {
   line-height: 24px;
   line-height: 2.4rem;
   letter-spacing: 2px;
-  text-transform: uppercase;
```
Comment (ojousima) [inline css/ruu-vi-styles.css]: Let's keep the uppercase

**Comment by ojousima** [comment]
Is the site available somewhere online for review?

**Comment by DG12** [comment]
http://mybeacons.info/index.1.1.3.html#AIgbAMLN

You might want to change 
To css/ruu-vi-styles.css .needsUpgrade  {font-family:sans-serif**;text-transform: none;**}
so entire sentence is no upper case.

**Comment by DG12** [comment]
By the way what do you use to edit html?

**Comment by DG12** [comment]
Restore transform:uppercase to &lt;P>, added transform:none to needsUpgrade.
I was waiting for Otso, Otso was waiting for me!
I'm still learning how GitHub is intended to work.

**Comment by ojousima** [comment]
Sorry for the delay. We'll look into ruu.vi (and associated issue with beacon ID missing on current Ruuvi FW) alongside with v1-backports release.

**Comment by DG12** [comment]
Have you decided not to use this?

**Pull request closed without merging**